lib/os.js

```diff
@@ -119,13 +119,13 @@ function loadavg() {
  * logical CPU core.
  *
  * @returns {Array<{
- *  model: string
- *  speed: number
+ *  model: string,
+ *  speed: number,
  *  times: {
- *    user: number
- *    nice: number
- *    sys: number
- *    idle: number
+ *    user: number,
+ *    nice: number,
+ *    sys: number,
+ *    idle: number,
  *    irq: number
  *  }
```

Comment:
Can we use trailing commas?
```suggestion
 *    irq: number,
 *  },
```

Reply:
we can